### ISSUE TITLE
Configurable keyword for the wildcard action and subject

### ISSUE THREAD

**Issue by stanleyk** [stalniy/casl#527]
**Is your feature request related to a problem? Please describe.**

Using the word "manage" as a wildcard alias for an action seems semantically incorrect in cases where it is not just about the simple "crud" actions. Any business domain oriented actions like "deliver", "pay" etc. in my opinion don't fit in what can be generalized as "manage".

**Describe the solution you'd like**

Since replacing the word "manage" with any other hardcoded alternative would cause an unappreciated BC break, I suggest introducing a configuration option to replace the default wildcard words with own alternatives. The configuration could be like this:

```
const ability = defineAbility({
    catchallAction: 'handle',
    catchallSubject: 'everything',
}, (can) => {
  can('handle', 'everything');
});
```

Thanks for considering this feature.

**Comment by stalniy** [comment]
hey, thanks for the suggestion. It makes sense can you make a PR for this?

**Comment by stalniy** [comment]
released in @casl/ability@5.4.0

Feel free to comment back any feedback in this issue
